frontend/src/utils/use_blur.ts

```diff
@@ -0,0 +1,22 @@
+import { useRef } from 'react';
+import { useEvent } from './use_event';
+
+export function useOnBlur(ref: HTMLElement | null, callback: () => void) {
```

Comment:
I think there is a better way... Will investigate this later today

Reply:
discussion moved here: https://github.com/transcribee/transcribee/pull/179#discussion_r1202632624